Source/WTF/wtf/PlatformHave.h

```diff
@@ -1904,3 +1914,13 @@
 #else
 #define HAVE_RE_STEREO_CONTENT_SUPPORT 0
 #endif
+
+#if !defined(HAVE_NWSETTINGS_UNIFIED_HTTP) && defined(NW_SETTINGS_HAS_UNIFIED_HTTP) && (!PLATFORM(MAC) || __MAC_OS_X_VERSION_MIN_REQUIRED >= 150000)
```

Comment:
NW_SETTINGS_HAS_UNIFIED_HTTP comes from Network.framework, so I'm pretty sure it won't work here, and `defined(NW_SETTINGS_HAS_UNIFIED_HTTP)` will always be false.

I would recommend seeing if we _really_ need to check that header, or if we can just use *_MIN_REQUIRED versions to figure out whether to enable this feature or not.